toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -37,7 +37,27 @@ const (
 	PackageManifestRelativePath = "image_pkg_manifest_installroot.json"
 
 	// NullDevice represents the /dev/null device used as a mount device for overlay images.
-	NullDevice     = "/dev/null"
+	NullDevice = "/dev/null"
+
+	// Kernel command-line args for enabling SELinux.
+	CmdlineSELinuxSettings = "security=selinux selinux=1"
+
+	// Kernel command-line args for enabling SELinux and force it to be in enforcing mode.
+	CmdlineSELinuxForceEnforcing = CmdlineSELinuxSettings + " enforcing=1"
+
+	// File path of the SELinux config file.
+	SELinuxConfigFile = "/etc/selinux/config"
+
+	// The valid string values for the "SELINUX" field in the /etc/selinux/config file.
+	SELinuxConfigEnforcing  = "enforcing"
+	SELinuxConfigPermissive = "permissive"
+	SELinuxConfigDisabled   = "disabled"
+
+	GrubCfgFile = "/boot/grub2/grub.cfg"
+	GrubDefFile = "/etc/default/grub"
```

Comment:
Thanks for organizing the constants! Could we add documentation for the `GrubCfg*` ones and start each documentation line with the name of the constant (similar to `NullDevice`)? We're tryint to follow [Golang's docs](https://go.dev/blog/godoc) about commenting the code.

Reply:
fixed